shared/libpng-util/Makefile

```diff
@@ -1,5 +1,9 @@
 .PHONY: libpng-util.a clean .FORCE
 
+CXXFLAGS+=-I/usr/local/include
```

Comment:
you should only need this once in config.mk inside an if bsd. And *either* once in shell's makeffile also inside an if bsd *OR* in your ey file for bsd